frontend/src/gamestate/experimentalGame.js

```diff
@@ -461,30 +498,29 @@ function MainGame() {
                         }
                     } else {
                         setShowWinnerOverlay(true);
-                        setTimeout(() => setShowWinnerOverlay(false), 3000); // Display winner overlay for 3 seconds
+                        reloadWithDelay(() => setShowWinnerOverlay(false));
                         return;
                     }
                 }
                 // Bot's turn to play
                 setCurrentTurn('bot');
 
-                setTimeout(()=>{
+                setTimeout(() => {
                     botPlayTurn(botData, tableData, setbotData, setTableData);
-                },1000)
-                if(botAmmount === 'twoBots' || botAmmount === 'threeBots'){
-                    setTimeout(()=>{
-                        botPlayTurn(botData2, tableData, setbotData2, setTableData);
-                    },2000)
+                }, 1000)
+                if (botAmmount === 'twoBots' || botAmmount === 'threeBots') {
+                    setTimeout(() => {
+                    }, 2000)
                 }
```

Comment:
You removed the botPlayTurn for the second bot!

Reply:
My bad, didn't realize. Added it back.

Reply:
Nice